fs/go.mod

```diff
@@ -1,6 +1,6 @@
 module darvaza.org/x/fs
 
-go 1.19
+go 1.21
```

Comment:
removing probably broken go 1.19 support?

Reply:
the github CI only tested 1.20+ ... that was a left over